core/domain/blog_domain.py

```diff
@@ -19,23 +19,50 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+import datetime
 import re
 
 from core import python_utils
 from core import utils
 from core.constants import constants
 from core.domain import html_cleaner
 
+from typing import Dict, List, Optional, Union
+import typing_extensions
```

Comment:
```suggestion
from typing_extensions import TypedDict
```

but you will need to add the `'typing_extensions'` to https://github.com/oppia/oppia/blob/develop/scripts/linters/pylint_extensions.py#L1372

Reply:
I did that first but there will be linting error. So this is the only option left.

Reply:
As I said you need to add the `'typing_extensions'` to https://github.com/oppia/oppia/blob/develop/scripts/linters/pylint_extensions.py#L1372 that will fix the linting error.

Reply:
Done.
Sorry about that. I actually missed that link.